modules/configuring-user-level-access.adoc

```diff
@@ -0,0 +1,47 @@
+// Module is included in the following assemblies:
+//
+// * installing-red-hat-openshift-gitops
+
+:_content-type: PROCEDURE
+[id="configuring-user-level-access_{context}"]
+= Configuring user level access
+
+[role="_abstract"]
+To manage and modify the user level access you need to configure the Argo CD RBAC configmap.
```

Comment:
This is possible now but we want to change this behavior in the future. We would like users to modify the Argo CD CR instead of Configmap to handle RBAC.

So please update the line to 
`To manage and modify the user level access you need to configure the RBAC section in Argo CD custom resource`